packages/plugin-dom-editable/src/EventNormalizer.ts

```diff
@@ -546,6 +534,22 @@ export class EventNormalizer {
         this._triggerEventBatch = null;
         this._isInEditable = null;
     }
+    /**
+     * When an event will happen outside the editor in concurency settings, we
+     * might need to capture the selection before the external event happen.
+     * Otherwise the external event could be triggered on a wrong selection.
+     * Also, we need to slick the `_currentStackObservation` to let characters
+     * typed after the external command being sent after the external command.
+     */
+    signalPostExternalEvent(): void {
```

Comment:
I don't understand what this function does. Neither by its name nor by its doc. It looks like the doc tries to explain "why" this function is useful but it fails to explain what it *does*.